### DISCUSSION TITLE
大家覺得申請實習的面試最需要注意甚麼?公司他們最在意哪方面的表現呢?

### DISCUSSION THREAD

**Discussion by Tchang323** [1101-GS4719-C/Python-Community-GS4719-C#3676] [General]
因為大四了，最近開始要申請一些實習，而面試是相對重要的一環，決定你能不能申請上，有些面試表現好的人或許能力比不上他人，但還是能順利通過，想問問大家對於面試的看法~~

**Comment by irenesunnn** [comment]
可以多參考校內或系上的資源，比自己去104投履歷的命中率高很多

**Comment by 113607514** [comment]
我最近也在思考申請實習的面試，感覺每個面試官都對不同的東西有不同的重視，但大部分的公司似乎最在意的還是應聘者的**態度與學習能力**。面試的時候，我覺得能夠表現出對工作內容的興趣和對公司文化的認同，是非常加分的。其實在準備面試時，我會特別強調自己對學習新知識的渴望，因為很多時候，尤其是實習生這樣的角色，並不會有非常多的經驗要求，但如果你能表現出能夠快速上手並且對挑戰保持積極態度，通常會讓面試官更有信心。除了這些，面試中的表現也很重要，比如溝通能力、解決問題的能力和團隊合作的精神，這些都能反映出一個人的整體素質。我自己每次面試後，都會反思自己是否有充分展示出這些特質，並且不斷調整自己的應對策略，因為每一次的面試，對我來說，都是學習和進步的機會。

**Comment by poyen0806** [comment]
根據我面試的情況來看，主要還是圍繞著你的個人處事態度還有你的履歷，會從履歷中延伸出很多問題，例如：拋出一堆問題核定履歷是否屬實、現場找你錯問你怎麼解決
